cluster/ci/origin/config/clone-source.yaml

```diff
@@ -51,12 +54,12 @@ objects:
     source:
       dockerfile: |
         FROM test-base:latest
-
+        ENV UMASK 0200
         ARG REPO_OWNER
         ARG REPO_NAME
         ARG PULL_REFS
 
-        RUN umask 0002 && release-ci cloneref --src-root=/data
+        RUN run.sh release-ci cloneref --src-root=/data
```

Comment:
why do we want this again? this makes re-using this for other projects that start with a different base hard, or at least means we need to move the `run.sh` into `openshift/origin-release`

Reply:
found this comment ... https://github.com/openshift/release/pull/309#discussion_r146331384

feel like we shouldn't bother with it